pkg/etcd/controller.go

```diff
@@ -0,0 +1,91 @@
+package etcd
```

Comment:
Is this controller an important backport or is it just along for the ride?

Reply:
i have no idea tbh, it had a change in your commit so it brought the whole file and i don't even know what it does at present i'm just trying to get wrangler. should it be dropped.

Reply:
https://github.com/k3s-io/k3s/pull/3565/commits/64f6e8a5828860d06e139436f245e3d34d3813c0#diff-cf89f4344b593fcd79120a77b5dd097bc9b740786d0f09b2596d103ab3214db6R8 this change seems to have made the cherry pick just add the whole file

Reply:
well it's looking pretty clear now by ci that it should be dropped. im going to drop it